### ISSUE TITLE
Reference to "aria-dialog" property

### ISSUE THREAD

**Issue by srbenoit** [w3c/html#1673]
HTML 5.2 and the current 5.3 Editor's Draft refer to the "aria-dialog" property here:
      https://w3c.github.io/html/dom.html#allowed-aria-roles-states-and-properties

There is no such property in WAI-ARIA - should this be "aria-modal"?

**Comment by chaals** [comment]
@scottaohara @stevefaulkner ?

**Comment by scottaohara** [comment]
yup, that should be `aria-modal`
PR incoming
